packages/gatsby-tinacms-git/gatsby-node.ts

```diff
@@ -32,9 +32,11 @@ exports.onCreateDevServer = (
     pathToContent,
     gitRemote,
     sshKey,
+    maxBuildTime,
     ...routerOptions
   } = options
 
+  process.env.MAX_BUILD_TIME = maxBuildTime || '1000'
```

Comment:
I don't think we should be here. The user should be responsible for setting their environment variables
```suggestion
```